elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -182,8 +182,14 @@ $pfe-tabs__tab--PaddingBottom-factor:  1.5; // 54px at 16px base.
   }
 }
 
-::slotted(h2) {
+::slotted(h1),
```

Comment:
Suggest:
```
%tab-headings {
    font-size: #{pfe-var(font-size)};
    font-weight: #{pfe-var(normal, font-weight)};
    margin: 0;
    user-select: none;
}

@for $i from 1 through 6 {
    ::slotted(h#{$i}) {
        @extend %tab-headings;
    }
}
```

Reply:
Hmm,  listing the headings might not seem DRY, but it is easier to scan quickly to find what you're looking for, because you can see the h1 or h2, etc. 
Since they both net out to 11 lines, can we leave the easier looking one for simplicity's sake?

Reply:
Sure, it's easier to read to me but I'm fine leaving it expanded if that's preferred.  Note that I also cleaned up the var references though too.  Could we pull that update in even if we don't use the loop?

Reply:
I will leave this as is then.